classes/fontproof.lua

```diff
@@ -57,11 +57,22 @@ function fontproof:init ()
   return plain.init(self)
 end
 
+local function getGitCommit()
+  -- If we are in a git repository, report the latest commit ID
+  local fh = io.popen("git describe --always --dirty='*'")
+  local commit = fh:read()
+  if commit then
+    return "("..commit..")"
```

Comment:
```suggestion
    return " ["..commit.."]"
```
Since there is already a paren string appended I think using brackets will help identify what data is what. Also the leading space because this is getting appended without further processing.